RecoTracker/TrackProducer/plugins/TrackExtraRekeyer.cc

```diff
@@ -0,0 +1,59 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+
+class TrackExtraRekeyer : public edm::stream::EDProducer<> {
```

Comment:
```suggestion
class TrackExtraRekeyer : public edm::global::EDProducer<> {
```

this can be a global module; the include and the methods change a bit